File: src/main/java/com/train/Ticket.java
package com.train;

public class Ticket {
    private int amount;
    private int price;
    private int roundTrip;

    public Ticket(int amount, int roundTrip) {
        this.amount = amount;
        this.roundTrip = roundTrip;
        setPrice((amount - roundTrip) * 1000 + (int) (roundTrip * 2000 * 0.9));
    }

    public int getAmount() {
        return amount;
    }

    public void setAmount(int amount) {
        this.amount = amount;
    }

    public String getPrice() {
        return "Total tickets: " + price;
    }

    public void setPrice(int price) {
        this.price = price;
    }

    public int getRoundTrip() {
        return roundTrip;
    }

    public void setRoundTrip(int roundTrip) {
        this.roundTrip = roundTrip;
    }
}

File: src/main/kotlin/com/train/Ticket.kt
package com

class Ticket(var amount: Int, var roundTrip: Int) {
    companion object {
        private const val ROUND_TRIP_PRICE = 2000
        private const val ONE_WAY = 1000
        private const val DISCOUNT = 0.9
    }

    fun getPrice() = "Total tickets: " + (amount - roundTrip) * ONE_WAY +
            (roundTrip.toDouble() * ROUND_TRIP_PRICE * DISCOUNT).toInt()
}